lib/get-prop.js

```diff
@@ -21,7 +21,18 @@ function getProp(key, pwd) {
   dec += cipher.final('utf8');
 
   // parse it
-  dec = JSON.parse(dec);
+  try {
+    dec = JSON.parse(dec);
+  } catch (ex) {
+    if (!item.backup) throw ex;
+
+    // attempt to use backup value if using an old key
+    cipher = crypto.createCipher(item.alg, pwd || this.pwd);
```

Comment:
It might make sense to move the decryption code into a function and parameterize the payload to avoid duplication.